src/lambda/explorer/lambdaFunctionNode.ts

```diff
@@ -9,6 +9,12 @@ import { getIcon } from '../../shared/icons'
 
 import { AWSResourceNode } from '../../shared/treeview/nodes/awsResourceNode'
 import { AWSTreeNodeBase } from '../../shared/treeview/nodes/awsTreeNodeBase'
+import { samLambdaImportableRuntimes } from '../models/samLambdaRuntime'
+
+export const CONTEXT_VALUE_LAMBDA_FUNCTION = 'awsRegionFunctionNode'
+export const CONTEXT_VALUE_LAMBDA_FUNCTION_IMPORTABLE = 'awsRegionFunctionNodeDownloadable'
+export const CONTEXT_VALUE_LAMBDA_FUNCTION_UPLOADABLE = 'awsRegionFunctionNodeUploadable'
+export const CONTEXT_VALUE_LAMBDA_FUNCTION_IMPORTABLE_UPLOADABLE = 'awsRegionFunctionNodeDownloadableUploadable'
```

Comment:
we've discussed in other PRs that we want to avoid uppercase names going forward. Since this is being changed in this PR, the names should be updated.

also in general, names this long should never exist. it is not necessary to have such long names. 

```suggestion
export const contextLambdaImportableUploadable = 'awsRegionFunctionNodeDownloadableUploadable'
```